packages/flutter_tools/test/general.shard/migrations/swift_package_manager_integration_migration_test.dart

```diff
@@ -2825,6 +3352,10 @@ String _projectIdentifier(SupportedPlatform platform) {
       : '33CC10E52044A3C60003C045';
 }
 
+String _relativeEphemeralPath(SupportedPlatform platform) {
+  return platform == SupportedPlatform.ios ? 'Flutter/ephemeral' : 'ephemeral';
```

Comment:
Would it be possible to use `XcodeBasedProject.ephemeralDirectory` instead and then convert that to a relative path? That'll make sure this logic doesn't break in the unlikely event we move the ephemeral directory. I don't feel strongly about this at all since this seems unlikely, just a thought.

Reply:
Whoops, I commented in the wrong code location. Let me move this...